smoke-tests/index.js

```diff
@@ -5,17 +5,24 @@ const { join, resolve } = require('path')
 const t = require('tap')
 const rimraf = promisify(require('rimraf'))
 
+const cwd = process.cwd()
+const npmDir = resolve(__dirname, '..', process.env.SMOKE_TEST_NPM_PACKAGE || '')
+
 const normalizePath = path => path.replace(/[A-Z]:/, '').replace(/\\/g, '/')
-const cwd = normalizePath(process.cwd())
+
 t.cleanSnapshot = s =>
   s
-    .split(cwd)
+    .split(normalizePath(npmDir))
+    .join('{CWD}')
+    .split(normalizePath(cwd))
     .join('{CWD}')
     .split(registry)
     .join('https://registry.npmjs.org/')
     .split(normalizePath(process.execPath))
     .join('node')
-    .split(process.cwd())
+    .split(npmDir)
```

Comment:
This feels very weird that it tries to do this twice, once on the normalized path and once on the real path?  At the least we should know why and put a comment in as to why it's intentional.

Reply:
Yeah, I'm not sure why it does that tbh. On posix systems those are the same, but on windows I think we sometimes print windows style paths and other times we print a normalized path?

Reply:
FUN